### DISCUSSION TITLE
tcp port(9000) and http port(8123) can be set as same number?

### DISCUSSION THREAD

**Discussion by flyly0755** [ClickHouse/ClickHouse#42881] [Q&A] [answered]
Now with such situation, in my project both use clickhouse_driver and clickhouse_sqlalchemy packages.
clickhouse_driver use tcp port 9000, clickhouse_sqlalchemy use port 8123.
usually clickhouse_driver use for raw sql select with better perfermence, and clickhouse_alchemy use for ORM coz of usability.
with clickhouse_alchemy ORM get the map raw sql language, and clickhouse_driver select data with this raw sql language.
so i want to change tcp port as same as http port, both use for example 8123, Is it feasible?

**Comment by mo-avatar** [comment, marked as answer]
To my knowledge, it's not feasible. please don't do that.
